jina/drivers/__init__.py

```diff
@@ -262,7 +266,7 @@ def __call__(self, *args, **kwargs):
     def _traverse_apply(self, docs, *args, **kwargs):
         """often useful when you delete a recursive structure """
 
-        def post_traverse(_docs, recur_on, context_doc=None):
+        def post_traverse(_docs, recur_on, context_doc=None, depth_name='granularity', recur_start=0, recur_end=1):
```

Comment:
Please update the docstring. Please put function arguments on new lines (line length = 115 is not `PEP8` conformant). I know we don't enforce `PEP8` too much until now, but I'd love to start a bit. Or, at some point in time, we just go to black.

Apart from the: Great refactoring of this function!

Reply:
I will do that for now, but yes I would love to have u help me enforce some style, I am not so experienced when it comes to Python standards.